docs/ja/docs/tutorial/static-files.md

```diff
@@ -0,0 +1,53 @@
+# 静的ファイル
+
+`StaticFiles` を使用して、ディレクトリから静的ファイルを自動的に提供できます。
+
+## `aiofiles` をインストール
+
+まず、`aiofiles` をインストールする必要があります:
+
+<div class="termy">
+
+```console
+$ pip install aiofiles
+
+---> 100%
+```
+
+</div>
+
+## `StaticFiles` の使用
+
+* `StaticFiles` をインポート。
+* `StaticFiles()` インスタンスを生成し、特定のパスに「マウント」。
+
+```Python hl_lines="2  6"
+{!../../../docs_src/static_files/tutorial001.py!}
+```
+
+!!! note "技術詳細"
+    `from starlette.staticfiles import StaticFiles` も使用できます。
+
+    **FastAPI**は、開発者の便利のために、`starlette.staticfiles` と同じ `fastapi.staticfiles` を提供します。しかし、実際にはStarletteから直接渡されています。
```

Comment:
```suggestion
    **FastAPI**は、開発者の利便性のために、`starlette.staticfiles` と同じ `fastapi.staticfiles` を提供します。しかし、実際にはStarletteから直接渡されています。
```

This might be more natural.